### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'last_backup_work_dir' when starting Zato server2

### ISSUE THREAD

**Issue by rafal-k** [zatosource/zato#730]
Yesterday I observed lots of AttributeErrors in server2's log during starting it. I'm attaching the log. [attributeerrors_server.txt](https://github.com/zatosource/zato/files/696161/attributeerrors_server.txt)

Today, however, I've only noticed one such AttributeError so far when starting the server:

```
2017-01-10 14:03:39,665 - INFO - 18834:MainThread - zato.server.base.parallel:155 - Found extra services to deploy: updates.check-updates
2017-01-10 14:03:39,678 - INFO - 18834:MainThread - zato.server.base.parallel:174 - Deployed an extra service found: updates.check-updates (434)
2017-01-10 14:03:39,683 - WARNING - 18834:DummyThread-1 - zato.server.service:394 - Traceback (most recent call last):
  File "/home/rafal/Projects/Zato/zato/code/zato-server/src/zato/server/service/__init__.py", line 382, in update_handle
    self._invoke(service, channel)
  File "/home/rafal/Projects/Zato/zato/code/zato-server/src/zato/server/service/__init__.py", line 320, in _invoke
    service.handle()
  File "/home/rafal/Projects/Zato/zato/code/zato-server/src/zato/server/service/internal/hot_deploy/__init__.py", line 213, in handle
    self.backup_current_work_dir()
  File "/home/rafal/Projects/Zato/zato/code/zato-server/src/zato/server/service/internal/hot_deploy/__init__.py", line 108, in backup_current_work_dir
    last_backup_work_dir = self.server.hot_deploy_config.last_backup_work_dir
AttributeError: 'NoneType' object has no attribute 'last_backup_work_dir'
```

Zato version:  `Zato 3.0.0rc1+rev.fb634a91`

**Comment by rafal-k** [comment]
I've just started server2 in a new, clean quickstart environment and I can see lots of AttributeErrors in the log again.

**Comment by dsuch** [comment]
Fixed in main since a868d4cf9475bad17bea68375633c06eac7699b0.
